apps/snitch_core/lib/core/data/schema/option_type.ex

```diff
@@ -0,0 +1,43 @@
+defmodule Snitch.Data.Schema.OptionType do
+  @moduledoc """
+  Models an OptionType
+  """
+  use Snitch.Data.Schema
+
+  alias Snitch.Data.Schema.TemplateOptionValue
+
+  @type t :: %__MODULE__{}
+
+  schema "snitch_option_types" do
+    field(:name, :string)
+    field(:display_name, :string)
+    field(:position, :integer)
```

Comment:
Please remove from here as well. I'm unable to see this page on my local because of this.